powerline.theme

```diff
@@ -437,4 +567,243 @@ __powerline_vcs_section() {
     fi
 }
 
-# vim: ft=sh et sts=4 sw=4 tw=120
+_lp_power2lines_theme_activate() {
+    _lp_powerline_plus_theme_activate
+}
+
+_lp_power2lines_theme_directory() {
+    _lp_powerline_plus_theme_directory
+}
+
+_lp_power2lines_theme_prompt() {
+    local lp_terminal_format
+    _lp_powerline_plus_theme_prompt
+    lp_terminal_format "${POWERLINE_LINE_COLOR[@]}"
+    PS1="${lp_terminal_format}╭╢${NO_COL}$PS1\n${lp_terminal_format}╰╴${NO_COL}"
```

Comment:
A string like `"\n"` _somehow_ works in Bash (it shouldn't), but it does not work in Zsh. That prints a literal `\n` in the prompt. This is a problem for powerpuff too.